src/CodeCompliance/Domain/Checks/NotUnique/ModuleName.php

```diff
@@ -46,6 +47,14 @@ public function getGuideline(): string
         return 'Module %s has to have project prefix, like %s%s.';
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentationUrl(): string
+    {
+        return static::DOCUMENTATION_BASE_URL . 'entity-name-is-not-unique.html#module-name-is-not-unique';
```

Comment:
looks like correct link
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.html#example-of-related-error-in-the-evaluator-output-module-name-is-not-unique

Reply:
In this case - we have a mistake in documentation at the moment. And the section for this error description has the wrong name. I fixed it inside of documentation - https://github.com/spryker/spryker-docs/pull/1262 . PR is waiting for approval.

So I put the address here that will be correct in the future :) At the moment the link for that is: 
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.html#constant-name-is-not-unique-1

So it contains the description and right after description - examples.